### ISSUE TITLE
Button focus indication is barely visible

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#11842]
## 📝 Provide detailed reproduction steps (if any)

1.  Open [full-featured editor](https://ckeditor.com/docs/ckeditor5/latest/examples/builds-custom/full-featured-editor.html).
2.  Navigate to Font size/Font family button using keyboard.
3.  Open the dropdown and navigate down/up using arrow keys.

### ✔️ Expected result

Selection is more visible.

### ❌ Actual result

Selection is barely visible.

![](https://user-images.githubusercontent.com/95343030/170883998-21fb3f1e-0c24-4edc-a175-58762719cfad.png)

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

## 📃 Other details

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
Couple of questions:

*   What's exactly wrong with the current focus indication? Is it too thin, is it a matter of color, is it a matter of conveying information by the color alone?
*   How should the improved focus indication look like?

**Comment by Comandeer** [comment]
[According to WCAG 2.1](https://www.w3.org/TR/WCAG21/#non-text-contrast), the contrast ratio between the unfocused state and the focused one should be at least 3:1. As the current focus indicator is drawn on top of of the dropdown's border, this ratio should be calculated against the border's color. And for now, [the contrast ratio is 2.08](https://contrast-ratio.com/#hsl%28208%2C79%25%2C51%25%29-on-%23c4c4c4) for the solid part of the indicator and [1.24 for the shadow](https://contrast-ratio.com/#%23bcdefb-on-%23c4c4c4).

The easiest solution would be to make both colors more contrasting, e.g. [switching to blue](https://contrast-ratio.com/#%2300ff-on-%23c4c4c4) for the shadow and [darker blue](https://contrast-ratio.com/#%230000aa-on-%23c4c4c4) for the solid part.

More info about focus indicators (although according to WCAG 2.2): https://www.sarasoueidan.com/blog/focus-indicators/

**Comment by mlewand** [comment]
@oleq can you have a look at it?
